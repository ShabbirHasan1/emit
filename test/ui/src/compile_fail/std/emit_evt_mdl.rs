fn main() {
    emit::emit!(
        evt: emit::evt!("template"),
        mdl: "test",
        "template",
    );
}
