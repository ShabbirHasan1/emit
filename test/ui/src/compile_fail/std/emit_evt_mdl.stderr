error: the `evt` argument cannot be set if any of `mdl`, `extent`, or `props` are also set
 --> src/compile_fail/std/emit_evt_mdl.rs:3:14
  |
3 |         evt: emit::evt!("template"),
  |              ^^^^^^^^^^^^^^^^^^^^^^
